packet/vlan.go

```diff
@@ -56,15 +56,15 @@ func (packet *Packet) GetEtherType() uint16 {
 		vptr := packet.unparsed()
 		vhdr := (*VLANHdr)(unsafe.Pointer(vptr))
 		return SwapBytesUint16(vhdr.EtherType)
-	} else {
-		return SwapBytesUint16(packet.Ether.EtherType)
 	}
+	return SwapBytesUint16(packet.Ether.EtherType)
+
```

Comment:
remove this line

Reply:
This line is necessary because swapping EtherType is now made in GetEtherType()

Reply:
Maybe. But empty line before the end of function cannot be necessary.